content/tracing/visualization/trace.md

```diff
@@ -52,8 +55,25 @@ When [submitting your traces][1] you can add attributes in the `meta` parameter.
 
 {{< img src="tracing/visualization/trace/trace_error_formating.png" alt="Error Formating" responsive="true" >}}
 
+{{% /tab %}}
+{{% tab "Host Info" %}}
+
+View the host information related to the trace including host tags and graphs marking the time of the trace.
+
+{{< img src="tracing/visualization/trace/trace_host_info.png" alt="Trace Host Info" responsive="true" style="width:90%;">}}
+
+{{% /tab %}}
+{{% tab "Logs" %}}
+
+See logs related to your service at the time of the trace. When you hover over a log, a line will display on the flame graph at the time of the log. If you click on the log, it links you to the [log explorer search][1].
```

Comment:
No future tenses in the doc 😄 

r/`When you hover over a log, a line will display on the flame graph at the time of the log.`/When you hover over a log, a line showing its timestamp is displayed on the trace flame graph.`